organisaatio-ui/src/tools/mappers.ts

```diff
@@ -11,3 +11,25 @@ export const mapLocalizedKoodiToLang = (lang: string, property: string, value: K
     value[property][lang] || value[property].fi || value[property].sv || value[property].en || '';
 
 //TODO tarttee katsoa ryhmissa tarvitaanko näitä vielä.
+
+export const mapVisibleKieletFromOpetuskielet = (opetuskieletOptions: KoodistoSelectOption[]): SupportedKieli[] => {
```

Comment:
tämä on aivan helkutin ruma -> voisi varmasti kirjoittaa paljon nätimmin / selkeämmin.
hint: `const result = new Set<string>();`

Reply:
tutkin ja kokeilen

Reply:
olisiko selkeämäpää jos tekisi jotenkin näin:
`
if (opetuskielet.includes('suomi')) {
  visibleKielet.add('fi')
}
if (opetuskielet.includes('ruotsi')) {
  visibleKielet.add('sv')
}
if (opetuskielet.includes('suomi/ruotsi')) {
visibleKielet.add('fi')  
visibleKielet.add('sv')
}
if (opetuskielet.includes('muu')) {
  visibleKielet.add('en')
}
if ( visibleKielet.size() === 0 ) {
  visibleKielet.add('fi')
}
return sorttaaJaMuljuttele(visibleKielet)
`